api_tests/package.json

```
@@ -42,16 +43,16 @@
         "chai-json-schema": "^1.5.0",
         "chai-string": "^1.5.0",
         "chai-subset": "^1.6.0",
-        "comit-sdk": "^0.15.0",
+        "comit-sdk": "file:.yalc/comit-sdk",
```

I guess that wasn't on purpose :D 
```suggestion
        "comit-sdk": "file:.yalc/comit-sdk",
```

---

Oh yeah, fixing this as soon as I tackle Thomas' suggestions!